### DISCUSSION TITLE
Folder for playlists

### DISCUSSION THREAD

**Discussion by spenc4r** [JunkFood02/Seal#155] [Q&A] [answered]
When downloading a playlist I think it would be convenient to make a subfolder inside the seal folder or at least have the option to

**Comment by JunkFood02** [comment, marked as answer]
> When downloading a playlist I think it would be convenient to make a subfolder inside the seal folder or at least have the option to

Download - Download directory - Custom subdirectory - edit "Path template" to `%(playlist)s/`

**Reply by Synthetic-Sage** [reply]
Could you explain by an example for me 
Like in my case it's (/storage/emulated/0/Download/Seal) as default so what I have to write if I want to save in sub folder
